groups/unifiedmanager/locals.tf

```diff
@@ -3,9 +3,8 @@
 # ------------------------------------------------------------------------
 locals {
   account_ids          = data.vault_generic_secret.account_ids.data
-  admin_cidrs          = values(data.vault_generic_secret.internal_cidrs.data)
+  admin_cidrs          = data.aws_ec2_managed_prefix_list.admin_cidr_ranges
```

Comment:
This is referencing the whole data object. You need to be referencing the `id` attribute and using that to form a list, as `ingress_prefix_list_ids` requires a list input.